commands/open-shell-container.ts

```diff
@@ -1,12 +1,20 @@
-import vscode = require('vscode');
+import * as vscode from 'vscode';
 import {ContainerItem, quickPickContainer} from './utils/quick-pick-container';
+import {DockerEngineType, docker} from './utils/docker-endpoint';
+
+const engineTypeShellCommands = {
+    [DockerEngineType.Linux]: "/bin/sh",
+    [DockerEngineType.Windows]: "cmd"
```

Comment:
here is the blog post: https://blogs.windows.com/windowsexperience/2016/11/17/announcing-windows-10-insider-preview-build-14971-for-pc/#hjIQ7kQcdSPrzb1z.97

Given this, i suggest we default to PowerShell instead of cmd.